### ISSUE TITLE
Optimize the API rate limits on the main screen

### ISSUE THREAD

**Issue by pixdemo** [KeypDemoCorp/awesome-ui#711]
Accessibility should be considered when implementing this.

The design mockups for this are available in the shared drive.
